### PR TITLE
Standby database creation with observer

### PR DESCRIPTION
**Pull request by abhisbyk**
Standby database creation, with option to enable observer in the DG configuration.

### CODE REVIEW

File: OracleDatabase/SingleInstance/dockerfiles/21.3.0/runOracle.sh [hunk 1]
```diff
@@ -111,6 +111,24 @@ trap _int SIGINT
 # Set SIGTERM handler
 trap _term SIGTERM
 
+# Observer only flow
+if [ "${OBSERVER_ONLY}" = "true" ]; then
+   export OBSERVER_BASE_DIR=${ORACLE_BASE}/oradata/
+   $ORACLE_BASE/$CREATE_OBSERVER_FILE $OBSERVER_NAME $PRIMARY_DB_CONN_STR $ORACLE_PWD $OBSERVER_BASE_DIR || exit 1;
+   if test [ ! -f "$OBSERVER_BASE_DIR/$OBSERVER_NAME/observer.log" ] then
```
Comment (Mahi0911) [inline OracleDatabase/SingleInstance/dockerfiles/21.3.0/runOracle.sh:118]: semicolon is missing after " test [ ! -f "$OBSERVER_BASE_DIR/$OBSERVER_NAME/observer.log" ] " , please checl

Comment (abhisbyk) [reply OracleDatabase/SingleInstance/dockerfiles/21.3.0/runOracle.sh]: Addressed in the following commit:
https://github.com/yunus-qureshi/docker-images/pull/18/commits/9a80cee7fb0d350fce054fef9bac3c9096e5a5a9

File: OracleDatabase/SingleInstance/dockerfiles/21.3.0/Dockerfile [hunk 2]
```diff
@@ -49,14 +49,19 @@ ENV ORACLE_BASE=/opt/oracle \
...
     SETUP_LINUX_FILE="setupLinuxEnv.sh" \
     CHECK_SPACE_FILE="checkSpace.sh" \
     CHECK_DB_FILE="checkDBStatus.sh" \
     USER_SCRIPTS_FILE="runUserScripts.sh" \
     INSTALL_DB_BINARIES_FILE="installDBBinaries.sh" \
     RELINK_BINARY_FILE="relinkOracleBinary.sh" \
     SLIMMING=$SLIMMING \
-    ENABLE_ARCHIVELOG=false
+    ENABLE_ARCHIVELOG=false \
+    CREATE_STDBY=false \
```
Comment (yunus-qureshi) [inline OracleDatabase/SingleInstance/dockerfiles/21.3.0/Dockerfile:61]: STANDBY_DB=false

File: OracleDatabase/SingleInstance/dockerfiles/21.3.0/Dockerfile [hunk 3]
```diff
@@ -49,14 +49,19 @@ ENV ORACLE_BASE=/opt/oracle \
...
+    PRIMARY_DB_CONN_STR="" \
+    OBSERVER_ONLY=false \
```
Comment (yunus-qureshi) [inline OracleDatabase/SingleInstance/dockerfiles/21.3.0/Dockerfile:63]: DG_OBSERVER_ONLY=false

File: OracleDatabase/SingleInstance/dockerfiles/21.3.0/Dockerfile [hunk 4]
```diff
@@ -49,14 +49,19 @@ ENV ORACLE_BASE=/opt/oracle \
...
+    OBSERVER_NAME=""
```
Comment (yunus-qureshi) [inline OracleDatabase/SingleInstance/dockerfiles/21.3.0/Dockerfile:64]: DG_OBSERVER_NAME

**Review by yunus-qureshi — APPROVED** [review]

**Pull request merged**